src/js/components/graph/indicator-overlay.jsx

```diff
@@ -0,0 +1,92 @@
+// This file defines the indicator overlay component - the popup that appears on
+//  on the device screen the first time a user's graph exceeds the max visible
+//  nodes, and illustrates the gesture to use to scroll the graph.
+import React from 'react'
+import Radium from 'radium'
+
+let IndicatorOverlay = React.createClass({
+
+  getInitialState() {
+    return {
+      visible: false // overlay is initially hidden
+    }
+  },
+
+  getStyles() {
+    return {
+      // CSS for the overlay
+      overlayContainer: {
+        position: 'fixed',
+        zIndex: 1300,
+        width: '100vw',
+        height: '100vh',
+        backgroundColor: '#000',
+        opacity: '0.7',
+        textAlign: 'center',
+        display: 'block',
+        transition: 'display 1s',
+        marginTop: '-112px'
+      },
+      // CSS for hidden overlay
+      overlayContainerHidden: {
+        display: 'none'
+      },
+      // white text/arrow illustration within overlay
+      indicatorImg: {
+        position: 'relative',
+        top: '80vh'
+      }
+    }
+  },
+
+  // hides overlay
+  hide() {
+    this.setState({visible: false})
+  },
+
+  // displays overlay, only when not already previously shown
+  show() {
+    if (!localStorage.getItem('indicator-overlay-drawn')) {
```

Comment:
I've been thinking about this for a bit, we should namespace the localstorage items.
Also the name could be a bit more declarative.

So I would suggest something like 'jolocom.onboarding.scrolling-indicator'